extra/resources/HealthCPUIoWait

```diff
@@ -0,0 +1,177 @@
+#!/bin/sh
+#
+# CPUIOWait RA
+# Measures CPU iowait % via top and writes #health-cpuiowait status into the CIB
+#
+# Originally based on ocf:pacemaker:HealthCPU RA
+# Copyright 2004-2009 SUSE LINUX AG, Lars Marowsky-Brée. All Rights Reserved.
+#
+# Later changes copyright 2009-2018 Michael Schwartzkopff
+# in collaboration with the Bull company. Merci! All Rights Reserved.
+#
+# Later changes copyright 2018 Maciej Sobkowiak <asm@egnyte.com>
+#
+# This source code is licensed under the GNU General Public License version 2
+# (GPLv2) WITHOUT ANY WARRANTY.
+#
+#######################################################################
+# Initialization:
+
+: ${OCF_FUNCTIONS=${OCF_ROOT}/resource.d/heartbeat/.ocf-shellfuncs}
+. ${OCF_FUNCTIONS}
+: ${__OCF_ACTION=$1}
+
+#######################################################################
+
+meta_data() {
+        cat <<END
+<?xml version="1.0"?>
+<!DOCTYPE resource-agent SYSTEM "ra-api-1.dtd">
+<resource-agent name="CPUIdle" version="1.0">
```

Comment:
The name here should match the file name

Reply:
Changed to CPUIoWait